mesonbuild/backend/vs2010backend.py

```diff
@@ -1117,7 +1117,11 @@ def gen_vcxproj(self, target, ofname, guid):
 
         # Add more libraries to be linked if needed
         for t in target.get_dependencies():
-            lobj = self.build.targets[t.get_id()]
+            if isinstance(t, build.CustomTargetIndex):
+                # We don't need the actual project here, just the library name
+                lobj = t
+            else:
+                lobj = self.build.targets[t.get_id()]
```

Comment:
@jpakkane , as far as I could understand, `self.build.targets[t.get_id()]` is used to get the target project in VS (in case dependency is just a single output of such project). But in that case I don't really see the need for `self.build.targets[t.get_id()]` here. Please correct me if my assumption is wrong.